### PR TITLE
contracts: Fix back to Normal Mode issue in multi-liquidations

### PR DESCRIPTION
**Pull request by bingen**
The initial fix was wrong, as now for case of ICR between 100% and
MCR, collToOffset would be zero, so it wouldn’t be subtracted.

Besides, subtracting the gas compensation too, which in that case was
missed too.

### CODE REVIEW

File: packages/contracts/contracts/TroveManager.sol [hunk 1]
```diff
@@ -477,15 +476,14 @@ contract TroveManager is LiquityBase, Ownable, CheckContract, ITroveManager {
     {
         singleLiquidation.entireTroveDebt = _entireTroveDebt;
         singleLiquidation.entireTroveColl = _entireTroveColl;
-        uint collToOffset = _entireTroveDebt.mul(MCR).div(_price);
+        uint collToLiquidate = _entireTroveDebt.mul(MCR).div(_price);
```
Comment (RickGriff) [inline packages/contracts/contracts/TroveManager.sol:479]: `collToLiquidate` here doesn't exclude the gas compensation. 

However in Normal Mode and the branches in Recovery Mode that don't do capped liquidation, `vars.collToLiquidate` does exclude the gas compensation.  Maybe we should make the naming consistent?

File: packages/contracts/contracts/TroveManager.sol [hunk 2]
```diff
@@ -721,7 +722,10 @@ contract TroveManager is LiquityBase, Ownable, CheckContract, ITroveManager {
                 // Update aggregate trackers
                 vars.remainingLUSDInStabPool = vars.remainingLUSDInStabPool.sub(singleLiquidation.debtToOffset);
                 vars.entireSystemDebt = vars.entireSystemDebt.sub(singleLiquidation.debtToOffset);
-                vars.entireSystemColl = vars.entireSystemColl.sub(singleLiquidation.collToOffset).sub(singleLiquidation.collSurplus);
+                vars.entireSystemColl = vars.entireSystemColl.
```
Comment (RickGriff) [inline packages/contracts/contracts/TroveManager.sol:725]: Cool, this is clearer now with three well-defined components

**Review by RickGriff — COMMENTED** [review]
Looks good - just a question around the use of the `collToLiquidate `name.

File: packages/contracts/contracts/TroveManager.sol (hunk 1 above)
Comment (bingen) [reply packages/contracts/contracts/TroveManager.sol:479]: Oh, I didn’t realize. Yes, sure, let’s change it here then (as it’s easier, it’s a new temp variable that only appears here). Suggestions? :wink:

**Review by RickGriff — APPROVED** [review]
Looks great!

**Pull request merged**